frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -156,7 +156,7 @@ class Sidebar extends Component<Props> {
       activeSidebar,
       sidebarStatus,
     } = this.props;
-    const hidden = ['forceHidden', 'forceCollapsed'].includes(sidebarStatus);
+    const hidden = sidebarStatus === 'forceHidden';
```

Comment:
If now there are only two possible values, that could be refactored to a bool? If it's the case, probably deserves a dedicated PR though..

Reply:
makes sense, but sidebarStatus is coming from custom store (decorator hoc) implementation which I don't want to touch and think we should remove completely.

https://github.com/digitalbitbox/bitbox-wallet-app/blob/0995f0a550384cbed44b8b15e0459a765b152c23/frontends/web/src/components/guide/guide.tsx#L37-L42